src/scoreSheetBot.py

```diff
@@ -1246,6 +1246,9 @@ async def flair(self, ctx: Context, member: discord.Member, *, new_crew: str = N
         if flairing_crew.overflow:
             overflow_server = discord.utils.get(self.bot.guilds, name=OVERFLOW_SERVER)
             of_after = set(overflow_server.get_member(member.id).roles)
+        flaired_crew = crew_members(flairing_crew, self)
+        if len(flaired_crew) == 40:
+            await ctx.message.author.send("You have just flaired the 40th person for your crew. When the first of the month hits, this will make you eligible for soft cap restrictions. Check out the SCS rules or use the <#492166249174925312> channel if you are unsure what this means.")
```

Comment:
Occasionally messages to people will fail , I'd recommend wrapping the message in a try block and pinging them if it fails. Something like
```
try:
    await ctx.author.send('...')
except discord.errors.Forbidden:
    await ctx.send(f'{ctx.author.mention}  You have just flaired the 40th person ....')
```